pom.xml

```diff
@@ -706,7 +706,7 @@
              be used even when compiling with java 1.7+ -->
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>animal-sniffer-maven-plugin</artifactId>
-        <version>1.11</version>
+        <version>1.15</version>
```

Comment:
My mistake -- undoing -- thought it was needed for annotation-based suppression (but I'm also about to undo that).